apm-sniffer/apm-sdk-plugin/jetty-plugin/jetty-client-9.x-plugin/pom.xml

```diff
@@ -31,7 +31,7 @@
     <url>http://maven.apache.org</url>
 
     <properties>
-        <jetty-client.version>9.1.0.v20131115</jetty-client.version>
+        <jetty-client.version>9.4.55.v20240627</jetty-client.version>
```

Comment:
Why change this? Would this break preview compatibility?

Reply:
because org.eclipse.jetty.client.api.Response.AsyncContentListener cound not be found in 9.1.x version
if the wrapper not implementing it, application will also get empty result when using jetty client 9.2.x or later, 

by testing, it is  compatible with preview version 9.1.0.v20131115
, 9.2.12.v20150709, 9.3.30.v20211001 and 9.4.55.v20240627

Reply:
If it doesn't exist in `9.1.0.v20131115`, how do we make sure of compatibility? This doesn't make sense.